src/test/Simulation.t.sol

```diff
@@ -557,7 +558,7 @@ contract SimulationTests is DSTest {
 
     function testTakeLinkV2NoProfit() public {
         (,,,, uint256 dustRad) = vat.ilks(linkName);
-        uint256 amountLink = dustRad / RAY;
+        uint256 amountLink = (dustRad / (getLinkPrice() * 10 ** 9)) * 2;
```

Comment:
since this is just a test helper function could you add:

```solidity
function getLinkPriceRad() {
  return getLinkPrice() * 10 ** 9;
}
```

then each of these could be:

```suggestion
        uint256 amountLink = (dustRad / getLinkPriceRad()) * 2;
```

Reply:
It would be `getLinkPriceRay`

Reply:
Done.